recipe/meta.yaml

```
@@ -1,36 +1,44 @@
 {% set name = "iminuit" %}
-{% set version = "1.5.4" %}
+{% set version = "2.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: f71535f3adc41cf9591505512e44b7392457c1d86cb77ac70ad7b045981d5c76
+- url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b0a3b8ac9dccadcb0f4554fc8fb9c4a3583f356a7071c37e17a0f2bd6cda1b88
+- git_url: https://github.com/scikit-hep/iminuit.git
+  git_rev:  v2.0.0
```

```suggestion
  git_rev:  v{{ version }}
```

Will change in next version - but we need a fix/patch for Windows builds anyway, so won't rerun CI just for this. If we also can get the tests added to the SDist, then we can drop this whole ugly addition. :)